node/src/node_test_utils.rs

```diff
@@ -307,6 +308,7 @@ pub fn make_stream_handler_pool_subs_from_recorder(addr: &Addr<Recorder>) -> Str
         bind: recipient!(addr, PoolBindMessage),
         node_query_response: recipient!(addr, DispatcherNodeQueryResponse),
         node_from_ui_sub: recipient!(addr, NodeFromUiMessage),
+        schedule_message_sub: recipient!(addr, MessageScheduler<DispatcherNodeQueryResponse>),
```

Comment:
This is not actually just `scheduled_msg` it should rather be `scheduled_node_query_response_sub`. Keep in mind that theoretically you could have multiple scheduled messages and the recipients for them in the same actor. So mentioning the message kind is important (by the way, it would be handy to be able to read it even if we were clearly sure it's gonna be the only scheduled message ever in this place).